### PR TITLE
Updated Installation Guide

### PR DESCRIPTION
**Pull request by roshnaeem**
**Description:** Updated the[ Installation guide](https://arviz-devs.github.io/arviz/getting_started/Installation.html) part. 

**Note**: In order to add `python>=3.6`, should I update the `requirements.txt` or add it separately here?

### CODE REVIEW

**Comment by aloctavodia** [comment]
adding it here is OK.

**Review by canyon289 — APPROVED** [review]

File: doc/source/getting_started/Installation.rst [hunk 1]
```diff
@@ -20,11 +20,11 @@ Using pip
 
     pip install arviz
 
-Use the below pip command to install ArviZ with all of it's :ref:`Optional-dependencies`.
+Use the below pip command to install ArviZ with all of its :ref:`Optional-dependencies`.
 
 .. code:: bash
 
-    pip install arviz[all]
+    pip install arviz all
```
Comment (OriolAbril) [inline doc/source/getting_started/Installation.rst:27]: This doesn't work for me at all. I think this change is motivated by zsh shells not working with `pip install arviz[all]` unlike bash shells that do work. I think the fix for that is `pip install "arviz[all]"` instead

**Review by OriolAbril — CHANGES_REQUESTED** [review]

File: doc/source/getting_started/Installation.rst (hunk 1 above)
Comment (roshnaeem) [reply doc/source/getting_started/Installation.rst]: Done.

Comment (OriolAbril) [reply doc/source/getting_started/Installation.rst]: just rechecked that and `pip install arviz[all]` doesn't work on a zsh shell. It has to be `pip install "arviz[all]"`

Comment (roshnaeem) [reply doc/source/getting_started/Installation.rst]: Yes, I tried it too. Changed, it is working. Thank you.

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**